docker-compose.yml

```
@@ -68,6 +69,7 @@ services:
       - -Ddatastore.oxalis.uri=webknossos:9000
     links:
       - fossildb-persisted:fossildb
+      - redis:redis
```

```suggestion
      - redis
```
This syntax is enough here, as the service-name and the dns entry match.